emr-etl-runner/emr.env.example

```diff
@@ -0,0 +1,77 @@
+EMR_ENABLE=true
+STORAGE_ENABLE=true
+AWS_SNOWPLOW_ACCESS_KEY=
+AWS_SNOWPLOW_SECRET_KEY=
+AWS_region_AWS=eu-west-1
+AWS_log_AWS=s3://MY-collector-logs/emrlogs
+AWS_S3_collector_AWS=s3://MYcollector-logs
+AWS_processing_AWS=s3://MY-collector-logs/processing
+AWS_archive_AWS=s3://MY-collector-archive/raw
+AWS_enriched_good_AWS=s3://MY-collector-out/enriched/good
+AWS_enriched_bad_AWS=s3://MY-collector-out/enriched/bad
+AWS_enriched_errors_AWS=s3://MY-collector-errors/enriched
+AWS_enriched_archive_AWS=s3://MY-collector-archive/enriched
+AWS_shredded_good_AWS=s3://MY-collector-out/shredded/good
+AWS_shredded_bad_AWS=s3://MY-collector-out/shredded/bad
+AWS_shredded_errors_AWS=s3://MY-collector-errors/shredded
+AWS_shredded_archive_AWS=s3://MY-collector-archive/shredded
+EMR_EC2_DefaultRole_EMR=EMR_EC2_DefaultRole
+EMR_DefaultRole_EMR=EMR_DefaultRole
+
+AWS_placement_AWS=
+AWS_ec2_subnet_id_AWS=
+
+AWS_ec2_key_name_AWS=
+
+DEBUG=true
+LOG_LEVEL=DEBUG
+#LOG_LEVEL=INFO
+AWS_app_id_AWS= MYplow
+AWS_collector_AWS=
+
+IGLU_REPO_NAME=
+IGLU_VENDOR_PREFIX=
+IGLU_URL=
+
+# Data storage section
+# uncommment the '*_ENABLE' statements of your choice of data storage
+# and fill in the other details
+
+# DYNAMODB_ENABLE=true
+DYNAMODB_SCHEMA_DATA_ACCESS_KEY_ID=$AWS_SNOWPLOW_ACCESS_KEY
+DYNAMODB_SCHEMA_DATA_SECRET_KEY_ID=$AWS_SNOWPLOW_SECRET_KEY
+DYNAMODB_SCHEMA_DATA_AWS_REGION=$AWS_region_AWS
+DYNAMODB_SCHEMA_DATA_TABLE=snowplow
+
+# ES_ENABLE=true
+ES_SCHEMA_DATA_HOST=es
+ES_SCHEMA_DATA_INDEX=
+ES_SCHEMA_DATA_TYPE=
+
+# PG_ENABLE=true
+PG_SCHEMA_DATA_HOST=pg
+PG_SCHEMA_DATA_DB=snowplowdb
+PG_SCHEMA_DATA_USER=snowplow
+PG_SCHEMA_DATA_PW=
+
+# REDSHIFT_ENABLE=true
+REDSHIFT_SCHEMA_DATA_HOST=
+REDSHIFT_SCHEMA_DATA_DB=
+REDSHIFT_SCHEMA_DATA_USER=
+REDSHIFT_SCHEMA_DATA_PW=
+
+# Enrichments
+# ANON_IP=true
+# API_REQUEST_ENRICHMENT=true
+# CAMPAIGN_ATTRIBUTION=true
+# COOKIE_EXTRACTOR=true
+# CURRENCY_CONVERSION=true
+# EVENT_FINGERPRINT_ENRICHMENT=true
+# HTTP_HEADER_EXTRACTOR=true
+# IP_LOOKUPS=true
+# JAVASCRIPT_SCRIPT_ENRICHMENT=true
+# REFERER_PARSER=true
+# SQL_QUERY_ENRICHMENT=true
+# UA_PARSER=true
+# USER_AGENT_UTILS=true
+# WEATHER_ENRICHMENT=true
```

Comment:
A few points on this approach:

- I think it's fine to let people fill the enrichments folder on their own and just have the folder as input
- do we want people editing this instead of the yaml they might be accustomed to?

What do you think?